app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute/Edit/Tab/Main.php

```diff
@@ -38,9 +38,8 @@ protected function _prepareForm()
     {
         parent::_prepareForm();
         $attributeObject = $this->getAttributeObject();
-        /* @var $form Varien_Data_Form */
         $form = $this->getForm();
-        /* @var $fieldset Varien_Data_Form_Element_Fieldset */
+        /* @var Varien_Data_Form_Element_Fieldset $fieldset */
```

Comment:
For VSCode:
```php
        /** @var Varien_Data_Form_Element_Fieldset $fieldset */
```

Reply:
Ahhhh ....

`/* @var` notation is wrong ... it is correct with two `*` .... found 1271 occurences in app/code/Mage ...

Is it okay to leave this for now and fix this in next PR?

Reply:
Not sure if i should love or hate you :P

Fix `/* @var` causes a plenty of new errors ... lets fix this in next PR :)

Reply:
Nice catch ... it would pop up further or later (and a view of these errors have been fix)

- `* @var $var Name` ... isn`t detected correctly
- `** @var $var Name` ... throws phpstan-errors
- `* @var Name @var` ... isn`t detected correctly
- `** @var Name @var` ... reveals follow-up errors ... here we go :smile: 

I LOVE THIS :heart: